### ISSUE TITLE
//cmd/gazelle:go_default_test is failing on master

### ISSUE THREAD

**Issue by jayconrod** [bazelbuild/bazel-gazelle#742]
```
    TestMatchProtoLibrary: integration_test.go:3010: proto/BUILD.bazel: got:
        load("@rules_proto//proto:defs.bzl", "proto_library")
        # gazelle:prefix example.com/foo
        
        some_rule(
            name = "gen_proto",
            out = "foo.proto",
        )
        
        proto_library(
            name = "existing_proto",
            srcs = ["foo.proto"],
        )
        
        want:
        
        load("@io_bazel_rules_go//go:def.bzl", "go_library")
        load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
        load("@rules_proto//proto:defs.bzl", "proto_library")
        # gazelle:prefix example.com/foo
        
        some_rule(
            name = "gen_proto",
            out = "foo.proto",
        )
        
        proto_library(
            name = "existing_proto",
            srcs = ["foo.proto"],
            visibility = ["//visibility:public"],
        )
        
        go_proto_library(
            name = "proto_go_proto",
            importpath = "example.com/foo",
            proto = ":existing_proto",
            visibility = ["//visibility:public"],
        )
        
        go_library(
            name = "go_default_library",
            embed = [":proto_go_proto"],
            importpath = "example.com/foo",
            visibility = ["//visibility:public"],
        )
--- FAIL: TestMatchProtoLibrary (0.00s)
```

Started on #730. I'll roll it back for now.

@linzhp, would you mind updating the test and send another PR? I think it just wasn't updated after #733.
